tmpl/blog/mirage-seal.md

```diff
@@ -0,0 +1,72 @@
+
+Log in to the web interface available through the registrar's website.  You can start the certificate signing process from the "services" tab, which exposes an "SSL" subtab.  Click that (Gandi doesn't need to know that we intend only to support TLS, not SSL).  Hit the "Get an SSL Certificate" button.  Standard SSL is fine.  Even if you're entitled to a free certificate, it will appear that you need to pay here; however at checkout, the total amount due will be 0 in your preferred currency.  Ask for a single address and, if you want to pay nothing, a valid period of 1 year.
+
+Copy the content of the certificate-signing request you generated earlier and paste it into the web form.  Gandi will also ask you to identify your TLS stack; unfortunately `ocaml-tls` isn't in the drop-down menu, so choose OTHER (and perhaps send them a nice note asking them to add the hottest TLS stack on the block to their list).  Click "submit" and click through the order form.
+
+From here on out, the process is automatic if you're buying a certificate for a domain you have registered through them, via the account with which you requested the certificate.  You should shortly receive an e-mail with a subject like "Procedure for the validation of your Standard SSL certificate", which explains the process in more detail, but really all you need to do is wait a while (about 30 minutes, for me).  After the certificate has been generated, Gandi will notify you by e-mail, and you can download your certificate from the SSL management screen.  Click the magnifying glass next to the name of the domain for which you generated the cert to do so.
+
+## Packaging Up a Site with Mirage-Seal
+
+Equipped with a private key and a certificate, let's make an HTTPS unikernel!  First, use `opam` to install `mirage-seal`.  (If `opam` or other MirageOS tooling aren't set up yet, check out the [instructions for getting started](http://openmirage.org/wiki/install).)
+
+```
+opam install mirage-seal
+```
+
+`mirage-seal` has a few required arguments.  (For more thorough documentation, see `mirage-seal --help` or [mirage-seal's README file](https://github.com/mirage/mirage-seal/blob/master/README.md).)
+
+* `--data`: one directory containing all the content that should be served by the unikernel.  Candidates for such a directory are the top-level output directory of a static site generator (such as `public` for octopress), the `DocumentRoot` of an Apache configuration, or the `root` of an nginx configuration.
+* `--keys`: one directory containing the certificate (`server.pem`) and key (`server.key`) for the site.
+
+To build a Xen unikernel, select the Xen mode with `-t xen` as well.  So in full:
+
+```
+mirage-seal --data=/home/me/coolwebsite/public --keys=/home/me/coolwebsite/secrets -t xen
```

Comment:
Called it out in the "more info available in `mirage-seal --help` bit.  I prefer to leave it to the tool's documentation because Only You Can Prevent Documentation Rot, but the section's already heavy on the particulars of invocation, so if others prefer to have it there, that's fine too.